### PR TITLE
go fmt

### PR DESCRIPTION
**Pull request by moshaad7**

### CODE REVIEW

File: geo/parse.go [hunk 1]
```diff
@@ -24,15 +24,24 @@ import (
 // interpret it is as geo point.  Supported formats:
 // Container:
 // slice length 2 (GeoJSON)
-//  first element lon, second element lat
+//
```
Comment (abhinavdangeti) [inline geo/parse.go:27]: Why the new empty line here, think you should undo this file's change and simply remove that white space before the text in the comment on line 27, 29, 31, 33 before running `go fmt ./...`

File: query.go [hunk 2]
```diff
@@ -48,7 +48,9 @@ func NewConjunctionQuery(conjuncts ...query.Query) *query.ConjunctionQuery {
 // NewDateRangeQuery creates a new Query for ranges
 // of date values.
 // Date strings are parsed using the DateTimeParser configured in the
-//  top-level config.QueryDateTimeParser
```
Comment (abhinavdangeti) [inline query.go:51]: ditto

File: query.go [hunk 3]
```diff
@@ -57,7 +59,9 @@ func NewDateRangeQuery(start, end time.Time) *query.DateRangeQuery {
 // NewDateRangeInclusiveQuery creates a new Query for ranges
 // of date values.
 // Date strings are parsed using the DateTimeParser configured in the
-//  top-level config.QueryDateTimeParser
```
Comment (abhinavdangeti) [inline query.go:60]: ditto

File: query.go (hunk 2 above)
Comment (moshaad7) [reply query.go:51]: This is for the benefit of [gopls](https://pkg.go.dev/golang.org/x/tools/gopls)

File: query.go (hunk 3 above)
Comment (moshaad7) [reply query.go:60]: This is for the benefit of [gopls](https://pkg.go.dev/golang.org/x/tools/gopls)

**Review by abhinavdangeti — APPROVED** [review]

**Pull request merged**